sdk/core/Azure.Core/tests/OperationInternalTests.cs

```diff
@@ -14,256 +15,113 @@
 
 namespace Azure.Core.Tests
 {
-    [TestFixture(true)]
-    [TestFixture(false)]
     public class OperationInternalTests
```

Comment:
Would it be possible to refactor the tests in a separate PR just to ensure we didn't regress anything? It's harder to tell when both the implementation and the tests all change at the same time.

Reply:
Unfortunately, some asserts in these were simply incorrect. For example, initial response should **not** be returned after `UpdateStatusAsync` when state isn't final. So I'm not sure what those tests have been validating exactly,